js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -226,6 +228,11 @@ const AssetGraphExplorerWithData: React.FC<
         ).join(',');
       }
 
+      const nextCenter = layout?.nodes[nextOpsNameSelection[nextOpsNameSelection.length - 1]];
```

Comment:
This kind of thing makes me wonder if we should make TS stricter about object/array access at some point. 🤔

Reply:
Yeah it wouldn't be a bad idea... just double checked that `layout?.nodes[undefined]` is safe and I think this is ok but it's definitely a little gross.